db/fns.md

```diff
@@ -1658,18 +1664,30 @@ where the page of results should begin, and must all be specified if any
 are specified.  Typically these values would be drawn from the last item
 in the previous page.
 
-### get_task_group
+### get_task_group_project_ids
+
+* *Mode*: read
+* *Arguments*:
+  * `task_group_id_in text`
+* *Returns*: `table`
+  * `project_id text`
+* *Last defined on version*: 81
+
+Get unique projectIds of the tasks in the task group
```

Comment:
In this case we should update description too

Reply:
this is the description. You mean name of the function?

Reply:
I think I meant that _if_ we change it so you can only list projectIds of sealed task groups, we should update this description to say _sealed_ task group. But, I think based on your other feedback, I'm tending to think that having multiple projects per taskgroup is getting a bit too complicated because of the potential race conditions, and if a task group has multiple projects, needing multiple seals also is messy (one per project). So that would break things too because everything expects zero or one seal per task group.

Reply:
yes, we have to discuss projectId idea further and see how we can improve it, or maybe get rid of it, since all the tasks existing in both deployments never set it! All tasks have "none" for projectId